socorro/processor/cache_manager.py

```diff
@@ -488,12 +488,21 @@ def _event_generator(self, nonblocking=False):
                     if is_verbose:
                         METRICS.gauge("usage", value=self.total_size)
 
-                    sorted_sizes = list(sorted(self.lru.values()))
-                    if sorted_sizes:
+                    if self.lru:
+                        sorted_sizes = list(sorted(self.lru.values()))
+                        avg = sum(sorted_sizes) / len(sorted_sizes)
+                        # Some metrics about file sizes
+                        METRICS.gauge("file_sizes.avg", avg)
                         METRICS.gauge("file_sizes.median", get_index(sorted_sizes, 50))
                         METRICS.gauge(
                             "file_sizes.ninety_five", get_index(sorted_sizes, 95)
                         )
+                        METRICS.gauge("file_sizes.max", sorted_sizes[-1])
+
+                        # Some metrics about what's in the cache
+                        METRICS.gauge("files.count", len(sorted_sizes))
+                        gt_500 = len([fs for fs in sorted_sizes if fs > 500_000_000])
+                        METRICS.gauge("files.gt_500", gt_500)
 
                     next_heartbeat = now + 1
```

Comment:
This is really spammy in a local dev environment and I don't think we need down-to-the-second granularity, so I'm going to change the interval to 60s.